show/main.py

```diff
@@ -1803,20 +1803,74 @@ def mmu():
 # 'reboot-cause' command ("show reboot-cause")
 #
 @cli.command('reboot-cause')
-def reboot_cause():
-    """Show cause of most recent reboot"""
-    PREVIOUS_REBOOT_CAUSE_FILE = "/host/reboot-cause/previous-reboot-cause.txt"
-
-    # At boot time, PREVIOUS_REBOOT_CAUSE_FILE is generated based on
-    # the contents of the 'reboot cause' file as it was left when the device
-    # went down for reboot. This file should always be created at boot,
-    # but check first just in case it's not present.
-    if not os.path.isfile(PREVIOUS_REBOOT_CAUSE_FILE):
-        click.echo("Unable to determine cause of previous reboot\n")
+@click.argument('history', required=False, type=click.Choice(["history"]))
+def reboot_cause(history):
+    """Show cause of reboot"""
+    REBOOT_CAUSE_DIR = "/host/reboot-cause/"
+    PREVIOUS_REBOOT_CAUSE_FILE = REBOOT_CAUSE_DIR + "last-reboot-cause.json"
```

Comment:
Prefer sticking with "previous-" rather than "last-", as it's a bit more specific, whereas "last" can have a few meanings.